### ISSUE TITLE
Stopping a print in MMU

### ISSUE THREAD

**Issue by TheSappy** [prusa3d/Prusa-Firmware#278]
It could be great if there was an option of stopping a print with MMU, that allows doing nothing with filament if you stop a print. After menu pops-up after stopping a print, I have to reset the printer, because I don`t want to take out filament. Usually, I stop a print because I don't like the quality of the print, adhesion or just experimenting.
But if I use reset function - I have to select filament, set temperature, etc.

**Comment by PavelSindler** [comment]
Yes it is on to do list and for me it has high priority. You cannot just do nothing with filament if you stop a print. Filament is in the nozzle when you are printing, but you want to "park" it the cooling tube and shape the tip of it before starting new print.

**Comment by Prusa-Support** [comment]
This issue will be closed due to it being obsolete. Please open a new issue if this is not solved yet

Marina Curzi
Prusa Research
